packages/client/src/v2-events/features/events/actions/print-certificate/pdfUtils.ts

```diff
@@ -148,22 +155,205 @@ export function compileSvg({
     }
   )
 
-  Handlebars.registerHelper('findUserById', function (u: User[], id: string) {
-    const user = u.find((usr) => usr.id === id)
+  Handlebars.registerHelper(
+    'findUserById',
+    function (id: string, propertyName: keyof User) {
+      const user = users.find((usr) => usr.id === id)
+      if (!user) {
+        return ''
+      } else if (propertyName === 'name') {
+        return getUsersFullName(user.name, 'en')
+      }
+      return user[propertyName]
+    }
+  )
+
+  /**
+   * Handlebars helper: resolveRefAndLookup
+   *
+   * Usage: {{resolveRefAndLookup 'child.address.other' 'district' 'town'}}
+   *
+   * This helper does two things:
+   * 1. Resolves the full object from the declaration using the provided property path.
+   * 2. Looks up a specific field (like 'district' or an alternative like 'town') within the resolved object.
+   *
+   * It's useful when the field is a reference (e.g., locationId, address.districtId type)
+   * and you want to extract one of its subfields with resolved value.
+   * If the resolved value is not an object or both fields are missing, it returns the raw value directly.
+   */
+  Handlebars.registerHelper(
+    'resolveRefAndLookup',
```

Comment:
The country implementor doesn't know about the refs that we use in the background or that it needs to be resolved. Those two are to do with our internal implementation.

Let's rename the function to be 

```handlebars
{{ formattedLookup $declaration "child.address.privateHome.state" }}
```

```handlebars
{{ lookup $declaration "child.birthLocation" }}
```

The first parameter is of course not really needed but it's a matter of consistency. 

This makes our own function it's a natural progression to the built-in one